docs/user/grid.md

```diff
@@ -0,0 +1,106 @@
+
+
+Currently, grid plots in pyDARN get the geomagnetic positions from the GRID file [`mlon` and `mlat`](https://pypi.org/project/aacgmv2/) that uses AACGMv2 coordinates. In the future, pyDARN will generate the geographical position, which will bring support for not standard gridded vector layouts.
+
+### Basic usage
+
+pyDARN and pyplot need to be imported, as well as any GRID file needs to be [read in](https://pydarn.readthedocs.io/en/master/user/SDarnRead/):
+
+```python
+import matplotlib.pyplot as plt
+import pydarn
+
+#Read in grid file using SuperDARDRead, then read_grid
+file = "path/to/grid/file"
+SDarn_read = pydarn.SuperDARNRead(file)
+grid_data = SDarn_read.read_grid()
+
+```
+With the grid data loaded as a list of dictionaries (`grid_data` variable in above example), you may now call the `plot_grid` method. Make sure you tell it which time or record (numbered from first recorded in file, counting from 0):
+```python
+import datetime as dt
+
+stime = dt.datetime(year, month, day, hour, minute)
+gridplot = pydarn.Grid.plot_grid(grid_data, start_time=stime)
+plt.show()
+
+```
+In this example, the record at `stime` was plotted with the defaulted parameter being gridded line-of-sight velocity:
+![](../imgs/grid_1.png)
+
+You might have noticed that the variable `gridplot` in the examples above actually holds some information. This contains the AACGM latitude and longitude of the gridded vectors plotted, as well as the data. If you instead change `gridplot` to 3 separate variables, it will return the latitude, longitude, and data info into separate variables:
+```python
+lats,lons,data=pydarn.Grid.plot_grid(grid_data, start_time=stime)
+```
+
+### Additional parameters
+
+In addition to gridded  line-of-sight velocity, you can choose one of three other data products to plot by setting `parameter=String name`:
+
+| Data product                          | String name |
+|---------------------------------------|-------------|
+| Line of sight velocity (m/s) [Default]| `vec`       |
+| Spectral width (m/s)                  | `wdt`       |
+| Power (dB)                            | `pwr`       |
+
+### Additional options
+
+Here is a list of all the current options than can be used with `plot_grid`
+
+| Option                         | Action                                                                                |
+| ------------------------------ | ------------------------------------------------------------------------------------- |
+| record=(int)                   | Record number to plot                                                                 |
+| start_time=(datetime.datetime) | The start time of the record to plot                                                  |
+| time_delta=(int)               | How close to the start time to be to the start time of the record                     |
```

Comment:
I don't really understand what this is describing, is it just shunting the start time along by dt? Or is this combined with record, so it's record 0 + a dt of 120 seconds for example?

Reply:
shunting 
like if you have a start time of 00:5 but the records  are every 2 minutes then it will take  00:04 at dt = 0 
however, if you have 10-minute spacing you may want to shunt to the closest 10 min record.